core/src/test/scala/com/github/johnynek/paiges/Generators.scala

```diff
@@ -0,0 +1,92 @@
+package com.github.johnynek.paiges
+
+import org.scalacheck.{Arbitrary, Cogen, Gen}
+
+object Generators {
+  import Doc.{ str, text }
+
+  val asciiString: Gen[String] =
+    for {
+      n <- Gen.choose(1, 10)
+      cs <- Gen.listOfN(n, Gen.choose(32.toChar, 126.toChar))
+    } yield cs.mkString
+
+  val generalString: Gen[String] =
+    implicitly[Arbitrary[String]].arbitrary
+
+  val doc0Gen: Gen[Doc] = Gen.frequency(
+    //(1, Doc.empty),
```

Comment:
should we uncomment now? I think `fill` was creating the tons of empty, right? Can we just add the change in `Doc.fill` where we check that we don't have an Empty separator?